src/main/java/uk/gov/justice/digital/domains/model/DomainDefinition.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.justice.digital.domains.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Use lombok for this class instead and consider renaming it to `Domain`.

See https://projectlombok.org/features/Data